frontend/src/app/App.tsx

```diff
@@ -131,9 +131,9 @@ const App = () => {
                   userPermissions={data.whoami.permissions}
                 />
                 <ProtectedRoute
-                  path={`/patients`}
-                  render={() => {
-                    return <ManagePatientsContainer />;
+                  path={`/patients/:page?`}
```

Comment:
I believe there is a convention of managing page numbers as query parameters as they are optional variables. 

Query parameters also allow for easier reading of the url (`?page=3` vs `/3`) and allows for the mixing and matching of filters, pagination and search (`?page=3&role=STAFF&daysSinceLastTest=10`)

Reply:
That makes sense, we just hadn't gotten there yet.

Reply:
As far as the convention though, more implementations seem to do it in the route than in query params. For our case we probably want to reset the page to 1 when the search changes, and we might as well change all of that in one place.

Reply:
To get this PR put to bed, I suggest we defer this until the next sprint when queries and searching are implemented.

Reply:
@TimBest Thoughts?

Reply:
+1 on pushing the fix off for another sprint. Please capture the work that still needs to be done in a ticket and attach it to the people pagination epic https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/747